database/migration/src/main/resources/changesets/replace_empty_custom_labels.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+
+    <changeSet author="rmunshi" id="replace_empty_custom_labels">
+        <update tableName="WORKFLOW_STORE_ENTRY">
+            <column name="CUSTOM_LABELS" type="LONGTEXT" value="{}" />
+            <where>CUSTOM_LABELS = ""</where>
+        </update>
+    </changeSet>
+
+    <changeSet id="custom_labels_not_null" author="rmunshi">
+        <addNotNullConstraint tableName="WORKFLOW_STORE_ENTRY"
+                              columnName="CUSTOM_LABELS"
+                              columnDataType="LONGTEXT"
+                              defaultNullValue="{}"/>
+    </changeSet>
+
+</databaseChangeLog>
```

Comment:
Today I Learned™: One can go back and modify _some_ parts of the changelog: http://www.liquibase.org/2009/03/what-effects-changeset-checksums.html

This helps my OCD RE: newlines at the end of files, but I don't think one will get away with changing other parts of the change set contents.